### ISSUE TITLE
I used '--harmony' flag to run my koa application but it reported error

### ISSUE THREAD

**Issue by hwoarangzk** [petruisfan/node-supervisor#152]
I wrote the following line to run my koa app:
``supervisor -w --harmony index.js``
But it reported:
```
function *respond(next) {
         ^
SyntaxError: Unexpected token *
    at exports.runInThisContext (vm.js:73:16)
    at Module._compile (module.js:443:25)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (C:\projects\html-file-server\release\index.js:4:9)
    at Object.<anonymous> (C:\projects\html-file-server\release\index.js:22:4)
    at Module._compile (module.js:460:26)
Sending SIGINT to child...
```
Is there anything I've done wrong? How to run ES6 app with supervisor correctly? The version of supervisor I'm using is `0.6.0`. Thanks for any advice

**Comment by mjseidel** [comment]
@hwoarangzk: I know this is a couple months old, but you need to switch the order of the flags, like this:

    supervisor --harmony -w index.js

The -w flag expects the filenames to follow immediately.

**Comment by hwoarangzk** [comment]
Thank you so much :) Problem solved

**Comment by Globik** [comment]
And how to enable supervisor with --harmony_destructuring flag??

**Comment by Globik** [comment]
Yes, i do. I have changed the code a little bit in a supervisor's file in the lib module. Added a variable of harmony_destructuring. Works like a charm)))

**Comment by ruffrey** [comment]
Any PR coming for `--harmony_destructuring` (j/w)?

**Comment by petruisfan** [comment]
Solved in v0.11.0 thx to @robwilkerson
